source/_integrations/renault.markdown

```diff
@@ -38,6 +38,8 @@ This integration provides the following platforms:
 
 All vehicles linked to the account should then get added as devices, with sensors added as linked entity.
 
+In some situation features may require subscription like *Pack EV Remote Control* and/or *Pack Smart Navigation*.
```

Comment:
I think it would be worth moving this into it's own section ## Prerequisites, right above the config flow.
I would make a suggestion, but it is too far outside the change